docs/usage_guide/advanced_usage/index.rst

```diff
@@ -33,11 +33,12 @@ If you are familiar with a concrete execution engine, it is possible to initiali
 Modin will automatically attach to it. Refer to :doc:`Modin engines </usage_guide/advanced_usage/modin_engines>` page
 for more details.
 
-Experimental APIs
------------------
-
-Modin also supports these experimental APIs on top of pandas that are under active development.
+Additional APIs
+---------------
 
+Modin also supports these additional APIs on top of pandas.
+- :py:meth:`~modin.pandas.DataFrame.modin.to_pandas` -- convert Modin DataFrame/Series to Pandas DataFrame/Series.
+- :py:meth:`~modin.pandas.DataFrame.modin.to_ray_dataset` -- convert Modin DataFrame/Series to Ray Dataset.
```

Comment:
Suggestions on how to make the documentation more user friendly are very welcome.